internal/server/iigointernal/utilities.go

```diff
@@ -41,6 +41,12 @@ func WithdrawFromCommonPool(value shared.Resources, gameState *gamestate.GameSta
 	}
 }
 
+func depositIntoClientPrivatePool(value shared.Resources, id shared.ClientID, state *gamestate.GameState) {
```

Comment:
It would be great if you had logs here. However considering that you'd need to access the server object to log correctly id say its a low priority fix.

Reply:
the logger should be passed in. @ me if you need help

Reply:
#229 - To be solved here